src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_ProhibitDTD.cs

```diff
@@ -234,10 +234,10 @@ public void v5(object param0, object param1)
         [Theory]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/30107")]
         //[Variation(Desc = "v6.2- Test Default value of ProhibitDTD for Add(XmlTextReader) for schema with DTD", Priority = 1, Params = new object[] { "bug356711_a.xsd" })]
-        [InlineData("bug356711_a.xsd")]
+        [InlineData("bug356711_a.xsd", 0)]
         //[Variation(Desc = "v6.1- Test Default value of ProhibitDTD for Add(XmlTextReader) with an xs:import for schema with DTD", Priority = 1, Params = new object[] { "bug356711.xsd" })]
-        [InlineData("bug356711.xsd")]
-        public void v6(object param0)
+        [InlineData("bug356711.xsd", 1)]
+        public void v6(object param0, object param1)
```

Comment:
you can pass the `int` directly here, the tests are old and got mostly auto-converted to XUnit, that's why they go through object

Reply:
I could but I'd prefer to do all the tests rather than just this one, is that what you want?

Reply:
either way works, treat as optional

Reply:
Ok, I decided to fix this one since I already changed this code, so it the args are now strongly typed.